src/main/java/org/prebid/server/bidder/minutemedia/MinuteMediaBidder.java

```diff
@@ -0,0 +1,125 @@
+package org.prebid.server.bidder.minutemedia;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.iab.openrtb.request.BidRequest;
+import com.iab.openrtb.request.Imp;
+import com.iab.openrtb.response.Bid;
+import com.iab.openrtb.response.BidResponse;
+import com.iab.openrtb.response.SeatBid;
+import org.apache.commons.collections4.CollectionUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.prebid.server.bidder.Bidder;
+import org.prebid.server.bidder.minutemedia.model.ExtImpMinuteMedia;
+import org.prebid.server.bidder.model.BidderBid;
+import org.prebid.server.bidder.model.BidderCall;
+import org.prebid.server.bidder.model.BidderError;
+import org.prebid.server.bidder.model.HttpRequest;
+import org.prebid.server.bidder.model.Result;
+import org.prebid.server.exception.PreBidException;
+import org.prebid.server.json.DecodeException;
+import org.prebid.server.json.JacksonMapper;
+import org.prebid.server.proto.openrtb.ext.ExtPrebid;
+import org.prebid.server.proto.openrtb.ext.response.BidType;
+import org.prebid.server.util.BidderUtil;
+import org.prebid.server.util.HttpUtil;
+
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+
+public class MinuteMediaBidder implements Bidder<BidRequest> {
+
+    private static final TypeReference<ExtPrebid<?, ExtImpMinuteMedia>> MINUTE_MEDIA_EXT_TYPE_REFERENCE =
+            new TypeReference<>() {
+            };
+    public static final String PUBLISHER_ID_MACRO = "{{PublisherId}}";
+
+    private final String endpointUrl;
+    private final JacksonMapper mapper;
+
+    public MinuteMediaBidder(String endpointUrl, JacksonMapper mapper) {
+        this.endpointUrl = HttpUtil.validateUrl(Objects.requireNonNull(endpointUrl));
+        this.mapper = Objects.requireNonNull(mapper);
+    }
+
+    @Override
+    public Result<List<HttpRequest<BidRequest>>> makeHttpRequests(BidRequest bidRequest) {
+        final String orgId;
+
+        try {
+            orgId = extractFirstImpOrdId(bidRequest.getImp());
```

Comment:
They are checking each imp for org, not just first

Reply:
Only first. They iterate over `request.imp` object, but they return right away.
```go
func extractOrg(openRTBRequest *openrtb2.BidRequest) (string, error) {
	var err error
	for _, imp := range openRTBRequest.Imp {
		var bidderExt adapters.ExtImpBidder
		if err = json.Unmarshal(imp.Ext, &bidderExt); err != nil {
			return "", fmt.Errorf("failed to unmarshal bidderExt: %w", err)
		}

		var impExt openrtb_ext.ImpExtMinuteMedia
		if err = json.Unmarshal(bidderExt.Bidder, &impExt); err != nil {
			return "", fmt.Errorf("failed to unmarshal ImpExtMinuteMedia: %w", err)
		}

		return strings.TrimSpace(impExt.Org), nil
	}

	return "", errors.New("no imps in bid request")
}
```

Reply:
I can't say that I like how it's done, but there was a discussion about it and they made an explicit decision to not do any kind of validation for this field...

Reply:
Agree, `org` is the required field and always will be here